doc/specs/vulkan/chapters/pipelines.txt

```diff
@@ -771,6 +771,18 @@ endif::VK_KHR_maintenance2[]
     The number of resources in pname:layout accessible to each shader stage
     that is used by the pipeline must: be less than or equal to
     sname:VkPhysicalDeviceLimits::pname:maxPerStageResources
+ifdef::VK_NV_clip_space_w_scaling[]
+  * If no element of the pname:pDynamicStates member of pname:pDynamicState
+    is ename:VK_DYNAMIC_STATE_VIEWPORT_W_SCALING_NV, and the
+    pname:viewportWScalingEnable, member of a
+    slink:VkPipelineViewportWScalingStateCreateInfoNV structure chained to
```

Comment:
I'd put a comma before "chained" here, making "chained to ... of pname:pViewportState" into a clause.

Reply:
I copied that. Comma is not used elsewhere there in the spec.